pkg/provider/aws/plugin.go

```diff
@@ -11,6 +11,62 @@ import (
 	"go.uber.org/zap"
 )
 
+// ExpandConstructs looks at all existing constructs in the construct graph and turns them into their respective AWS Resources
+func (a *AWS) ExpandConstructs(result *core.ConstructGraph, dag *core.ResourceGraph) (err error) {
+	log := zap.S()
+	constructIds, err := result.TopologicalSort()
+	if err != nil {
+		return
+	}
+	// We want to reverse the list so that we start at the leaf nodes. This allows us to check downstream dependencies each time and process them.
```

Comment:
Is this true? I thought this stage doesn't do any cross construct dependencies (as that's done in the `CopyConstructEdgesToDag` step)

Reply:
yeah removed, good catch